database/connect.go

```diff
@@ -0,0 +1,32 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/probe-nitt/probe-server/config"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+// ConnectDB connect to db
+func ConnectDB() {
+
+	config := config.GetConfig()
+
+	var err error
+
+	dsn := config.DBUsername + ":" + config.DBPassword + "@tcp(" + config.DBHost + ":" + config.DBPort + ")/" + config.DBName + "?charset=utf8mb4&parseTime=True&loc=Local"
```

Comment:
https://freshman.tech/snippets/go/string-concatenation/
above doc has some other efficient ways of concatenating strings in go
kindly refer to it.